docs/week-5/exercises.md

```diff
@@ -1,12 +1,12 @@
 ## 15 Data Points
-Did you know that [with just 15 data points, researchers were able to identify 99.98% of Americans](https://www.reddit.com/r/privacy/comments/cjbo37/using_15_data_points_researchers_can_identify/)? What 15 data points do you think can identify you?
+Did you know that [with just 15 pieces of personal information, researchers were able to identify 99.98% of Americans](https://www.reddit.com/r/privacy/comments/cjbo37/using_15_data_points_researchers_can_identify/)? What 15 pieces of personal information do you think can identify you?
```

Comment:
but the post said data points...

Reply:
Don't believe everything you read on the internet. :smile: 
The posts' use of the term data point is [wrong](https://en.wikipedia.org/wiki/Unit_of_observation#Data_point) and can lead to confusion in the context of your exercise. 
See: